storybook/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -46,10 +46,17 @@ export const MultipleDescriptions: Story = {
   args: {
     children: [
       <DescriptionList.Term key={1}>Blinde, slechtziende</DescriptionList.Term>,
-      <DescriptionList.Description key={2}>Persoon met een visuele beperking</DescriptionList.Description>,
-      <DescriptionList.Description key={3}>Persoon met een visuele handicap</DescriptionList.Description>,
-      <DescriptionList.Description key={4}>Persoon die blind is</DescriptionList.Description>,
-      <DescriptionList.Description key={5}>Persoon die slechtziend is</DescriptionList.Description>,
+      <DescriptionList.Description key={1.2}>Persoon met een visuele beperking</DescriptionList.Description>,
+      <DescriptionList.Description key={1.3}>Persoon met een visuele handicap</DescriptionList.Description>,
+      <DescriptionList.Description key={1.4}>Persoon die blind is</DescriptionList.Description>,
+      <DescriptionList.Description key={1.5}>Persoon die slechtziend is</DescriptionList.Description>,
+      <DescriptionList.Term key={2}>Doof</DescriptionList.Term>,
+      <DescriptionList.Term key={3}>Slechthorend</DescriptionList.Term>,
+      <DescriptionList.Description key={2.1} style={{ gridRowStart: 5 }}>
```

Comment:
Well you actually only need to supply a row number for the tablet breakpoint, you need to "auto" rows part to work otherwise.

Reply:
Also the column count for grid goes up to 12, which is managable in CSS. If you have a description list of 50+ items it gets pretty bulky.

Reply:
> Well you actually only need to supply a row number for the tablet breakpoint, you need to "auto" rows part to work otherwise. Also the column count for grid goes up to 12, which is managable in CSS. If you have a description list of 50+ items it gets pretty bulky.

I’m not sure I understand. Let’s talk this through, unless the `div` approach is a good solution.

> A more pragmatic approach would be to wrap multiple terms in a div.

Interesting. I thought all related `dt` and `dd` elements should be wrapped in one `div`. You only wrap the `dt`s and leave the `dd` as a sibling. Is that valid? Does it work correctly in assistive technologies?

I consulted the [WHATWG spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element).

The content model seems to require the former approach:

> Content model:
> - Either: Zero or more groups each consisting of one or more dt elements followed by one or more dd elements, optionally intermixed with script-supporting elements.
> - Or: One or more div elements, optionally intermixed with script-supporting elements.

While this information allows the latter as well:

> The dl element represents an association list consisting of zero or more name-value groups (a description list). A name-value group consists of one or more names (dt elements, possibly as children of a div element child) followed by one or more values (dd elements, possibly as children of a div element child), ignoring any nodes other than dt and dd element children, and dt and dd elements that are children of div element children.

Please test whether wrapping only the `dt`s in a `div` is valid HTML and whether screen readers announce the terms and descriptions correctly.

Reply:
You're right, not valid html, which is a bummer.

Reply:
I've push a solution using groups. Unfortunately `subgrid` would be very useful here but the group class has been added to the various modifiers that set the width of the first column.

Only caveat is that it does need a `minWith` prop set to `md` to look any good.